mamba/mamba/utils.py

```diff
@@ -26,6 +26,19 @@
 
 import libmambapy as api
 
+try:
+    from conda.models.enums import PackageType
+except ImportError:
+    PackageType = None
+
+
+def _is_virtual_system(rec):
+    if PackageType is None:
+        # Backward compat for older conda versions
+        return rec.package_type == "virtual_system"
```

Comment:
Based on my latest comment above, I suppose that this is not needed and we could always import the `PackageType` enum instead. Let me know what you think.